plans/worker-package/worker-framework.md

```
@@ -0,0 +1,247 @@
+| Concern                                                                     | Owner                                          |
+| --------------------------------------------------------------------------- | ---------------------------------------------- |
+| Concrete service RPC definitions (`ClientServicesRpcs`, IdentityService, …) | `@dxos/client-protocol`, `@dxos/protocols/rpc` |
+| Service host implementations (`WorkerRuntime`, `ClientServicesHost`)        | `@dxos/client-services`                        |
+| System-port WebRTC bridge (`SharedWorkerConnection`)                        | `@dxos/client`                                 |
+| App-port service proxy (`ClientServicesProxy`)                              | `@dxos/client`                                 |
+| Low-level msgpack framing over `RpcPort`                                    | `@dxos/rpc` (`effect-rpc.ts`)                  |
+| SharedWorker-based client services (non-dedicated path)                     | `@dxos/client` (`WorkerClientServices`)        |
+
+## Package layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)